eclass/qt5-build.eclass

```diff
@@ -179,8 +179,14 @@ qt5-build_src_prepare() {
 		qt5_symlink_tools_to_build_dir
 
 		# Avoid unnecessary qmake recompilations
-		sed -i -re "s|^if true;.*(\[ '\!').*(\"\\\$outpath/bin/qmake\".*)|if \1 -e \2 then|" \
-			configure || die "sed failed (skip qmake bootstrap)"
+		if [[ ${QT5_MINOR_VERSION} -ge 8 ]]; then
+			sed -i -e '/Creating qmake/i if [ '!' -e "$outpath/bin/qmake" ]; then' \
```

Comment:
hmm there's a quoting problem here, I'm afraid you have to use double quotes as outermost quotes (for the whole sed expression) and escape them when used as literal quotes around `$outpath/bin/qmake`. Also don't forget to escape the `$`.